__tests__/DMT.test.js

```diff
@@ -0,0 +1,346 @@
+import { TransformationsHandler } from '../utils/DMTHandler';
```

Comment:
The failure paths in the test case execution are not tested throughout this file.